### ISSUE TITLE
{Freeway-Bug}[Adaptive Cards> Expenses Report]: No name is defined for the 'Show History' button.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8972]
### Target Platforms

UWP

### SDK Version

1.0.1.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Universal
OS Build: 24H2 (OS Build 26252.5000)
App Version: 1.0.1.0
Screen Reader: Narrator

**Repro Steps:**
1. Open the Adaptive Cards Universal application.
2. Adaptive Card UWP Visualizer screen starts appearing.
3. Navigate to Expenses Report.Json button by using Tab key and activate it.
4. Expenses Report adaptive card will appear on right sight of the pane.
5. Navigate to Expenses Report card by using Tab key.
6. Press the tab key and navigate to the 'Show History' button and observe the issue.

**Actual Result:**
While navigating the expenses report card, screen reader is not conveying the name of the 'Show History' button as no name is defined for the same. When the focus lands on it then it is announcing only as 'Button.'

Also, after activating the 'Show History' button screen reader is not conveying the changed name information. It is remain silent.

**Expected Result**
Name should be defined for the Show History button so that when the screen reader focus lands on it then it should announce the information as 'Show History button.'
Also, when user activates show history button then it should announce as 'Hide History' and vice versa.

**Note:**
Same issue is repro with NVDA.

**User Impact:**
This will impact the screen reader user as they will not be able to know the name of the control due to this they will not able to get to know by activating the control what action will be performed.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section.

### Screenshots

![No name is defined for the 'Show History' button](https://github.com/user-attachments/assets/b1d58e8d-0a77-43a9-90a1-128e1f16576e)

https://github.com/user-attachments/assets/63fb3bed-be91-46b3-ba56-e656e4462f1f

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
This issue is still repro, hence we will reactivate this bug in next month first week.

**Build Version:**
1.0.2.0

**Snippet:**
![Image](https://github.com/user-attachments/assets/171359f6-9ac6-47e1-bd0a-b587b8744ca5)

**Comment by vagpt** [comment]
Now this issue is being tracking with below bug URL hence closing this issue.

https://github.com/microsoft/AdaptiveCards/issues/9076
